From 581a25d6dd543e56cb0d3291031818acac28999b Mon Sep 17 00:00:00 2001
From: Georg Koppen <gk@torproject.org>
Date: Thu, 8 Oct 2020 08:22:52 +0000
Subject: [PATCH] Update glean for tor-browser-build#40085


diff --git a/buildSrc/src/main/java/Dependencies.kt b/buildSrc/src/main/java/Dependencies.kt
index cb56fbbcf..3d9580f51 100644
--- a/buildSrc/src/main/java/Dependencies.kt
+++ b/buildSrc/src/main/java/Dependencies.kt
@@ -29,7 +29,7 @@ object Versions {
 
     const val mozilla_appservices = "63.0.0"
 
-    const val mozilla_glean = "32.4.0"
+    const val mozilla_glean = "32.4.1"
 
     const val material = "1.1.0"
     const val nearby = "17.0.0"
-- 
2.28.0

